var/spack/repos/builtin/packages/vtkm/package.py

```diff
@@ -49,6 +51,12 @@ class Vtkm(CMakePackage, CudaPackage):
     def cmake_args(self):
         spec = self.spec
         options = []
+        gpu_name_table = {'20': 'fermi',
+                          '30': 'kepler',  '32': 'kepler',  '35': 'kepler',
+                          '50': 'maxwell', '52': 'maxwell', '53': 'maxwell',
+                          '60': 'pascal',  '61': 'pascal',  '62': 'pascal',
+                          '70': 'turing',  '72': 'turing',  '75': 'turing'}
+        print(gpu_name_table)
```

Comment:
Nitpick: is this important to keep, or a remnant of debugging?

Reply:
Also, do you think that gpu mapping table could be useful beyond just `vtk-m`?  If so, it could made into a module-level variable where `CudaPackage` is defined.  Personally, I wouldn't hold this up over it though.

Reply:
I scanned the spack packages. It looks like that Kokkos has something similar, but with more fine grained names. So in short, I don't see a strong case for propagating this up currently.